site/content/en/docs/reference/kubectl-kueue/commands/list.md

```diff
@@ -16,6 +16,8 @@ kubectl kueue list [TYPE]
 ### Examples:
 
 ```bash
+# List cluster queues
+kubectl kueue list clusterqueue
```

Comment:
Don't forget to update [here](https://github.com/kubernetes-sigs/kueue/blob/92ea2f1ab93d54e7403504170fda84343b5e3518/site/content/en/docs/reference/kubectl-kueue/_index.md?plain=1#L7).